pom.xml

```diff
@@ -76,6 +76,12 @@
             <version>0.0.5</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
```

Comment:
Sorry, I only brought it into test scope for convenience methods to verify serialization and deserialization. I can remove the dep and inline the serialization/deserialization code in the test.